app/web/features/communities/events/EventPage.tsx

```diff
@@ -261,7 +260,10 @@ export default function EventPage({
                     >
                       {t("communities:virtual_event")}
                     </Typography>
-                    <MuiLink href={event.onlineInformation.link}>
+                    <MuiLink
+                      href={event.onlineInformation.link}
+                      underline="hover"
```

Comment:
Links don't underline on hover by default anymore, so we need to add that if we want it. I noticed we had it, so I ran that codemod.